MobileSDK/sdk/src/main/java/org/lantern/mobilesdk/Lantern.java

```diff
@@ -6,26 +6,17 @@
 import com.google.android.gms.analytics.HitBuilders;
 import com.google.android.gms.analytics.Tracker;
 
-
 import java.io.File;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
- * Created by ox.to.a.cart on 1/28/16.
+ * API for embedding the Lantern proxy
  */
-public class Lantern {
+public abstract class Lantern {
```

Comment:
Yeah, that works. I wanted to see how Lantern as a service works with FireTweet since there's been instability issues in the past too. I also think the functionality to configure a webview to use our proxy is really useful and should be part of the SDK but we can add that back in a separate PR. 

One other thing I noticed is that FireTweet is using an old version of Google's analytics library, and integrating the SDK actually caused the app to crash until I upgraded for whatever reason. It's working fine now though and I've uploaded the new version here: https://raw.githubusercontent.com/firetweet/downloads/master/firetweet.apk